ldapadmin/src/main/webapp/console/app/components/user/user.html

```diff
@@ -20,41 +20,115 @@ <h3 class="panel-title">{{::user.user.sn}} {{::user.user.givenName}}</h3>
+                    <input ng-model="user.user.mail" type="email" class="form-control" id="email" placeholder="Email">
+                  </div>
+                </div>
+                <div class="form-group form-group-sm">
+                  <label class="col-sm-4" for="address">Address</label>
+                  <div class="col-sm-8">
+                    <input ng-model="user.user.postalAddress" class="form-control" id="address" placeholder="Address">
+                  </div>
+                </div>
+
+                <div class="form-group form-group-sm">
+                  <label class="col-sm-4" for="organization">Organization</label>
+                  <div class="col-sm-8">
+                    <input ng-model="user.user.o" class="form-control" id="organization" placeholder="Organization">
+                  </div>
+                </div>
+
+                <div class="form-group form-group-sm">
+                  <label class="col-sm-4" for="organization">Description</label>
+                  <div class="col-sm-8">
+                    <input ng-model="user.user.description" class="form-control" id="description" placeholder="Description">
+                  </div>
+                </div>
+                <div class="form-group form-group-sm">
+                  <label class="col-sm-4" for="manager">Nom du Responsable</label>
+                  <div class="col-sm-8">
+                    <input ng-model="user.user.manager" class="form-control" id="manager" placeholder="Manager">
+                  </div>
+                </div>
+                <div class="form-group form-group-sm">
+                  <label class="col-sm-4" for="shadowExpire">shadowExpire</label>
+                  <div class="col-sm-8">
+                    <input ng-model="user.user.shadowExpire" class="form-control" id="shadowExpire" placeholder="shadowExpire">
+                  </div>
+                </div>
+                <div class="form-group form-group-sm">
+                  <label class="col-sm-4" for="preferredLanguage">Contexte</label>
+                  <div class="col-sm-8">
+                    <input ng-model="user.user.preferredLanguage" class="form-control" id="preferredLanguage"
+                    placeholder="preferredLanguage">
+                  </div>
+                </div>
+
+                <button ng-click="user.save()" type="submit" class="btn btn-primary">Enregistrer</button>
+              </form>
             </div>
-            <div class="form-group">
-              <label class="col-sm-3" for="email">Email address</label>
-              <div class="col-sm-9">
-                <input ng-model="user.user.mail" type="email" class="form-control" id="email" placeholder="Email">
-              </div>
-            </div>
-            <button ng-click="user.save()" type="submit" class="btn btn-primary">Enregistrer</button>
-          </form>
 
+            <div class="col-sm-6">
+              <form class="form-horizontal ">
```

Comment:
Last comment, it would be better to englobe both columns with a single form, rather 1 form per column.